plugins/src/core/ddl.ts

```diff
@@ -0,0 +1,10 @@
+interface DeepLinkContent {
+  title?: string;
+  description?: string;
+}
+
+export interface DeepLink {
+  content: DeepLinkContent;
+  data: any;
```

Comment:
data is a json object, mb can be a bit more specific with type

Reply:
![image](https://user-images.githubusercontent.com/106683327/191010139-05303a3c-0ac2-426a-9303-f8f8fae469b3.png)
 I took this object as a reference, but of course when I look at the actual object that we send from the native function the objects are different, I'll fix this now.

Reply:
resolution can be an enum